src/main/java/org/alliancegenome/curation_api/model/entities/HighThroughputExpressionDatasetAnnotation.java

```diff
@@ -0,0 +1,60 @@
+package org.alliancegenome.curation_api.model.entities;
+
+import java.util.List;
+
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.view.View;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.search.mapper.pojo.automaticindexing.ReindexOnUpdate;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexingDependency;
+
+import com.fasterxml.jackson.annotation.JsonView;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Indexed
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@ToString(callSuper = true)
+@Schema(name = "HighThroughputExpressionDatasetAnnotation", description = "POJO that represents the HighThroughputExpressionDatasetAnnotation")
+public class HighThroughputExpressionDatasetAnnotation extends SubmittedObject {
+
+	@JsonView({ View.FieldsOnly.class })
```

Comment:
Need annotations here to index the name field - they are absent for the name field of BiologicalEntity subclasses because there is a bridge that defines the name index for these classes.  An example of the annotations for a string field can be seen in the PhenotypeAnnotation entity class.

Reply:
Updated this by indexing using fulltextfiled and keywordfield annotations